mk/artifact.mk

```diff
@@ -149,14 +163,26 @@ fetch-checksum:
 ifeq ($(call has, PREBUILT), 1)
 	$(Q)$(PRINTF) "Fetching SHA-1 of prebuilt binaries ...\n"
 ifeq ($(call has, SYSTEM), 1)
-	$(Q)wget -q -O $(BIN_DIR)/sha1sum-linux-image $(PREBUILT_BLOB_URL)/sha1sum-linux-image
-	$(Q)$(call notice, [OK])
+    ifeq ($(wildcard $(BIN_DIR)/rv32emu-linux-image-prebuilt.tar.gz),)
+		wget -q -O $(BIN_DIR)/sha1sum-linux-image $(PREBUILT_BLOB_URL)/sha1sum-linux-image
+		$(Q)$(call notice, [OK])
+    else
+		$(info Downloading skipped.)
+    endif
 else ifeq ($(call has, ARCH_TEST), 1)
-	$(Q)wget -q -O $(BIN_DIR)/rv32emu-prebuilt-sail-$(HOST_PLATFORM).sha $(PREBUILT_BLOB_URL)/rv32emu-prebuilt-sail-$(HOST_PLATFORM).sha
+    ifeq ($(wildcard $(BIN_DIR)/rv32emu-prebuilt-sail-$(HOST_PLATFORM)),)
+		wget -q -O $(BIN_DIR)/rv32emu-prebuilt-sail-$(HOST_PLATFORM).sha $(PREBUILT_BLOB_URL)/rv32emu-prebuilt-sail-$(HOST_PLATFORM).sha
+    else
+		$(info Downloading skipped.)
+    endif
 else
-	$(Q)wget -q -O $(BIN_DIR)/sha1sum-linux-x86-softfp $(PREBUILT_BLOB_URL)/sha1sum-linux-x86-softfp
-	$(Q)wget -q -O $(BIN_DIR)/sha1sum-riscv32 $(PREBUILT_BLOB_URL)/sha1sum-riscv32
-	$(Q)$(call notice, [OK])
+    ifeq ($(wildcard $(BIN_DIR)/rv32emu-prebuilt.tar.gz),)
+		wget -q -O $(BIN_DIR)/sha1sum-linux-x86-softfp $(PREBUILT_BLOB_URL)/sha1sum-linux-x86-softfp
+		wget -q -O $(BIN_DIR)/sha1sum-riscv32 $(PREBUILT_BLOB_URL)/sha1sum-riscv32
```

Comment:
Later, we may consider to introduce [function calls](https://www.gnu.org/software/make/manual/html_node/Call-Function.html) to wrap `wget` in another pull request.

Reply:
Sure. Moreover, consolidating the file name of sha1sum and prebuilts, such as `rv32emu-prebuilt-*.sha` would make it easier.